pkg/test/framework/components/istio/ingress.go

```diff
@@ -232,39 +234,37 @@ func (c *ingressImpl) Call(options ingress.CallOptions) (ingress.CallResponse, e
 	if err := options.Sanitize(); err != nil {
 		scopes.Framework.Fatalf("CallOptions sanitization failure, error %v", err)
 	}
-	client, err := c.createClient(options)
-	if err != nil {
-		scopes.Framework.Errorf("failed to create test client, error %v", err)
-		return ingress.CallResponse{}, err
+	req := &echo.CallOptions{
+		Port: &echo.Port{
+			Protocol:    protocol.HTTP,
+			ServicePort: options.Address.Port,
+		},
+		Host: options.Address.IP.String(),
+		Path: options.Path,
+		Headers: map[string][]string{
+			"Host": {options.Host},
+		},
+		CaCert: options.CaCert,
```

Comment:
Are we dropping the cert/key pair here?

Reply:
Also, until we fully convert this including the callers, maybe if CallType is PlainText we should strip CaCert/Cert/Key from the request, in-case callers have those set alongside PlainText for whatever reason (convenience maybe ??) - I guess that's really only necessary if you see related failures.

Reply:
Probably, good catch - I think that's why the tests are failing. I also didn't implement the CallType == mtls so that may cause some additional failures.